neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
Nějaký typ výjimky, který má cenu chytat. Chytání výjimek se dělá jen v případě, že se z toho dokáže ten program vzpamatovat. Chytnout výjimku, jenom kvůli tomu, abys mohl něco vyblít na konzoli a skončit je prostě zlo (kór když součástí toho výblitku je i stack trace). Je tam modul s výjimkama (`config.exceptions`), kde se dá v klidu přidat nový typ výjimky, která před svojí message  vypíše ještě "failed to load ini file:", a nemusí se tim prasit kód jinde.